apps/remix-ide/src/app/files/fileManager.ts

```diff
@@ -217,6 +218,38 @@ class FileManager extends Plugin {
     }
   }
 
+  /**
+  * Set the content of multiple files
+  * @param {string[]} filePaths paths of the files
+  * @param {string[]} data content to write to each file
+  * @param {string} folderPath base folder path
+  * @returns {void}
+  */
+  async writeMultipleFiles(filePaths, fileData, folderPath) {
+    try {
+      let alert = true
+      for (let i = 0; i < filePaths.length; i++) {
+        let installPath = folderPath + "/" + filePaths[i]
```

Comment:
Since installPath is not modified any where else. This can be a const rather than a let.

Reply:
Updated installPath to const rather than let